### PR TITLE
redirect unsigned user appropriate post after viewmore clicked

### LINKED ISSUE #1203: Unsigned user not redirected to post page clicked on
Environment:Staging
1.As a unsigned user click on help board.
2.Click on long post(see more).Directs user to sign in page.
3.User logs in by entering valid username ,valid password.
Expected Result:
User has to land on the particular long post.
Actual Result:
User lands on home page.

### PR DESCRIPTION
**Pull request by OzkanAbdullahoglu**
### What does this pull request aim to achieve? 💯
Resolves #1203 
_Please be concise and link any related issue(s):_

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [x] This branch is rebased/merged with the **latest master**.
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

**Comment by joshmorel** [comment]
Hey @OzkanAbdullahoglu I'll take a look locally.

@idjevm I think it didn't build because this is behind master and there was some change to terraform files in the interim. Anyways no big deal.

**Comment by idjevm** [comment]
@joshmorel Yeah since this is not priority for soft launch, let's have @OzkanAbdullahoglu  refactor a bit! We already spoke and he did mention that he will be taking a look to combine these into a single func.

**Comment by OzkanAbdullahoglu** [comment]
I believe now It is better?

File: client/src/components/Feed/Post.js [hunk 1]
```diff
@@ -325,6 +325,9 @@ const Post = ({
           <ViewMore onClick={onClick} loadContent={loadMorePost} />
         ) : (
           <Link
+                onClick={() =>
+                  sessionStorage.setItem("postredirect", `/post/${post._id}`)
+                }
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:330]: Just some formatting here, but everything else looks good to go!

**Pull request merged**